packages/web/docs/src/app/case-studies/(posts)/sound-xyz/page.mdx

```diff
@@ -0,0 +1,148 @@
+---
+title: 'Sound.xyz: Scaling GraphQL Infrastructure for Web3 Music Innovation'
+excerpt:
+  'Sound.xyz is revolutionizing the music industry by addressing two critical challenges: the
+  concentration of streaming revenue among top artists and the inadequate compensation per stream.'
+category: Music
+authors:
+  - name: Author
+    position: Position
+    avatar: https://placedog.net/24/24
+---
+
+import { DocsIcon, LargeCallout } from '#components/large-callout'
+import { Lede } from '#components/lede'
```

Comment:
Let me know if you think this is useful in global MDX scope. Same for `LargeCallout`.